packages/appsync-modelgen-plugin/src/__tests__/visitors/appsync-swift-visitor.test.ts

```diff
@@ -275,6 +275,48 @@ describe('AppSyncSwiftVisitor', () => {
     `);
   });
 
+  it('Should handle nullability of lists appropriately', () => {
+    const schema = /* GraphQL */ `
+      enum StatusEnum {
+        pass
+        fail
+      }
+
+      type CustomType {
+        name: String
+        list: [Int]
+        requiredList: [String]!
+        requiredListOfRequired: [StatusEnum!]!
+        listOfRequired: [Boolean!]
+      }
+
+      type ListContainer
+      @model
+      {
+        id: ID!
+        name: String
+        list: [Int]
+        requiredList: [String]!
+        requiredListOfRequired: [StatusEnum!]!
+        listOfRequired: [Boolean!]
+        requiredListOfRequiredDates: [AWSDate!]!
+        listOfRequiredFloats: [Float!]
+        requiredListOfCustomTypes: [CustomType]!
+      }
+    `;
+
+    const visitor = getVisitor(schema, 'ListContainer');
+    const generatedCode = visitor.generate();
+    expect(generatedCode).toMatchSnapshot();
+
+    const metadataVisitor = getVisitor(schema, 'ListContainer', CodeGenGenerateEnum.metadata);
+    const generatedMetadata = metadataVisitor.generate();
+    expect(generatedMetadata).toMatchSnapshot();
+
+    const customTypeVisitor = getVisitor(schema, 'CustomType');
+    expect(customTypeVisitor.generate()).toMatchSnapshot();
```

Comment:
The swift test is using the inline snapshot for now. Although it is better to use a separate file for snapshot, we should keep all of them in one place. I suggest to change all inline snapshots into one file or keep this test as inline temporarily and move them in another PR.

Reply:
Keeping snapshots separate is preferred over inline snaps. I am not sure why we should keep all inline snaps for now?

Reply:
What I mean is that we should keep the snapshots into one place. I agree it is better to keep them in a separate file.However, you just add the new case in snapshot file but keep the old ones as inline.